libs/journeys/ui/src/components/SearchBar/SearchDropdown/RefinementGroups/RefinementGroups.tsx

```diff
@@ -40,14 +40,13 @@ export function RefinementGroups({
 }: RefinementGroupsProps): ReactElement {
   const { t } = useTranslation('apps-watch')
   const theme = useTheme()
+  const [isOpen, setIsOpen] = useState(false)
```

Comment:
Would this be a better name?
```suggestion
  const [isCollapsed, setIsCollapsed] = useState(false)
```